scripts/install/deluge.sh

```diff
@@ -245,26 +249,25 @@ WantedBy=multi-user.target
 DW
   fi
 for u in "${users[@]}"; do
-  systemctl enable deluged@${u} >>"${log}" 2>&1
-  systemctl enable deluge-web@${u} >>"${log}" 2>&1
+  systemctl enable -q deluged@${u} 2>&1  | tee -a $log
```

Comment:
I'm assuming `-q` means quiet? What is the point of teeing to the log here?

Reply:
`-q` = no output when there's no error. When there's error, show it to user, and then append it to the log as well so it shows up in both places.